recipes/lazrs-python/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "lazrs-python" %}
```

Comment:
Any reason this has not the same name as on PyPI?

Reply:
The original Rust implementation is called laz-rs (see https://github.com/laz-rs/laz-rs). Presumably that would be on conda-forge one day too.